examples/svm/plot_custom_kernel.py

```diff
@@ -46,7 +46,7 @@ def my_kernel(X, Y):
 
 # Put the result into a color plot
 Z = Z.reshape(xx.shape)
-plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired)
+plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired, shading="nearest")
```

Comment:
On `main`, this is now calling `DecisionBoundaryDisplay.from_estimator`, with `shading="auto"`, which results in very similar plots.